improver_tests/acceptance/SHA256SUMS

```diff
@@ -164,15 +164,15 @@ fe84db49d69ea62e2e82312c7b3d860c7329c627d79d76c2543d365f69d2e0d8  ./estimate-emo
 fc0d25a60a0ab753d019310c0e718110a6d994b2e8e6147e6262a0e6eea794e8  ./extract/basic/kgo.nc
 925c95be9546a6e3034fa50a80b61a5330fa1e6e1ecd24ae31e0adca5eb0cddf  ./extract/change_units/kgo.nc
 9db96e63e4235d410941deadc16ed9608df91f33ddf7656acd9fa3971e9ff3bc  ./extract/grids/input_grid_latlon.nc
-073bec02083dfaacf432d544a9ea9faace10a0c614f3425be408d22310789570  ./extract/grids/input_grid_uk.nc
+6af6442eba0ecc2079c7ad850606254cb5ce15f0b584a9d93821b098d2c6081f  ./extract/grids/input_grid_uk.nc
 7b3ba137fa37b1e18a70f48529c0e5c36feb013a8e4b2e5cafbdd8afb1c04fd2  ./extract/grids/kgo_grid_latlon.nc
-bb62f5fdea51b59294983d01469bf8a00b30728645472086d0d37e6b961e52a6  ./extract/grids/kgo_grid_uk.nc
+2b2792de8bdb6e22cf0a6e0d748d9c55e4099f1ed0cdcdc843cd10cf88014a15  ./extract/grids/kgo_grid_uk.nc
 29c314c8f0890f7697d51996b8623533b90f06979af11ebe58a015c7b82a584e  ./extract/list_constraints/kgo.nc
 f7b0393c1a87cf966bd665635fd65226b780ff9f95bf6a13946e73ae2a417f7b  ./extract/multiple_constraints/kgo.nc
 f7b0393c1a87cf966bd665635fd65226b780ff9f95bf6a13946e73ae2a417f7b  ./extract/multiple_constraints_units/kgo.nc
 2293a8fd52d235746cd755f2177e7d0f869d3b49158c593f6c6e64baa08435ad  ./extract/range_constraints/kgo.nc
-090633826529fa4786a4a7dacfe60312b9ee0817bd0d0935dd830540c8c52128  ./extract/sites/input_spot.nc
-700ae5fb088a0db121dba089c9838740229c917a031873b8835e87b84d245e89  ./extract/sites/kgo_spot.nc
+c2abfc4f83d4a918dd2d80f20b7746e5ae19d809d21df9715bd8ed5011d38ed2  ./extract/sites/input_spot.nc
```

Comment:
In this file, the `wmo_id` units have been updated from `no_unit` to `unknown`. Note that this is similar to the issues encountered in #1634.  Possibly using improver's save_netcdf, rather than Iris save, might correct this.

Reply:
This is standard Iris 3 behaviour. The improver save wrapper does not change it. The real-time PS45 suites are also producing output with the wmo_id unit set to "unknown".